### PR TITLE
fix naming consistency

### PR DESCRIPTION
**Pull request by mshaban-msft**
### Changelog
- renaming 'classification' to 'class' in classification result
- renaming 'CustomSingleClassificationTask' to 'CustomSingleLabelClassificationTask'
- renaming 'CustomMultiClassificationTask' to 'CustomMultiLabelClassificationTask'

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json [hunk 1]
```diff
@@ -215,8 +215,8 @@
         "Healthcare",
         "ExtractiveSummarization",
         "CustomEntityRecognition",
-        "CustomSingleClassification",
-        "CustomMultiClassification"
+        "CustomSingleClass",
```
Comment (kayousef-zz) [inline specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json]: mmmmm, I'm not sure if we want to change this or not. I was thinking of the list name only, not the task name.

Comment (mshaban-msft) [reply specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json]: that seems counter-intuitive!
you want to unify the naming across all services, but at the same time, same api has 2 different conventions for both request and responses?!

**Comment by abhahn** [comment]
Hi @mshaban-msft , could you move your updates to the textanalytics.json file in the 2022-03-01-preview folder?

File: specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json (hunk 1 above)
Comment (abhahn) [reply specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json]: The reason for the different names in request and responses has to do with the use of the discriminator pattern.  The discriminator actually needs different names here in order to differentiate the schemas.  My preference would have been to have the names the same, but without the difference the Swagger definition will not pass model validation.

File: specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json [hunk 2]
```diff
@@ -779,7 +779,7 @@
     "MultiClassificationDocumentResult": {
       "type": "object",
       "properties": {
-        "classifications": {
+        "classs": {
```
Comment (abhahn) [inline specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json]: I think there is an extra `s` added in this property name.

File: specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json [hunk 3]
```diff
@@ -792,7 +792,7 @@
         }
       ],
       "required": [
-        "classifications"
+        "classs"
```
Comment (abhahn) [inline specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/textanalytics.json]: Here as well.

**Pull request merged**